src/app/components/index/index-footer/index-footer/index-footer.component.html

```diff
@@ -0,0 +1,32 @@
+<footer>
+  <table mdbTable borderless="true" id="facebookTable">
```

Comment:
In my opinion it's better to use classes, as it's more clear and id only if you refer to specific elemen from ts

Reply:
Class added